src/Contrib/composer.json

```diff
@@ -13,6 +13,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
+        "ext-sockets": "*",
```

Comment:
I think it should only be a suggest...if you're not using thrift, you don't need it. However, it could be added to our dev Dockerfile

Reply:
Moved it to a suggest here too. And I think Timo added it to the dev image for me a while back so we should be good there